raiden/transfermanager.py

```diff
@@ -68,11 +85,15 @@ def transfer_async(self, amount, identifier, target, callback=None):
             timeout.
         """
 
+        # Create a default identifier value
+        if not identifier:
```

Comment:
well...screw my comment. In this context `if not identifier` seems to be the better comparison here, since other falsy/empty values (`[], 0, ()`...) would also not work as an identifier, even if supplied by the user.